components/search_box.js

```diff
@@ -53,7 +53,8 @@ const ClearButton = styled("button")({
   backgroundRepeat: "no-repeat",
   backgroundPosition: "2px 50%",
   ":focus": {
-    outlineOffset: 0
+    outlineOffset: 0,
+    outline: `3px solid ` + globalTheme.colour.focusColour
```

Comment:
We could move the entire  \`3px solid\` + globalTheme.colour.focusColour into a `globalTheme.focusOutline` if we wanted.

Reply:
that's a good idea. want to make an issue for it? :)